src/app/projects/components/audit-log/audit-log.component.ts

```diff
@@ -0,0 +1,28 @@
+import {formatDate} from "@angular/common";
+import {Component, Input} from '@angular/core';
+import {AuditLog} from "@projects/models/audit-log";
+import { lowerCase, startCase } from "lodash";
+
+@Component({
+  selector: 'app-audit-log',
+  templateUrl: './audit-log.component.html',
+  styleUrls: ['./audit-log.component.css']
+})
+
+export class AuditLogComponent {
+  @Input()
+  auditLogs: AuditLog[];
+
+  getAuditEvent(auditLog: AuditLog) {
+    return `${auditLog['auditType']} from ${this.formatAuditEvent(auditLog['before'])} to ${
+      this.formatAuditEvent(auditLog['after'])}`;
+  }
+
+  formatAuditDate(date: string): string {
```

Comment:
This kind of formatting is usually done in the html template rather than in code.
See https://www.npmjs.com/package/ngx-pipes